package.json

```diff
@@ -42,6 +43,11 @@
   "author": "Teachiq",
   "license": "ISC",
   "eslintConfig": {
-    "extends": "./index.js"
+    "extends": [
+      "./profile/vue.js",
+      "./profile/vue3.js",
+      "./profile/vueTs.js",
```

Comment:
Can we name vueTs to vue-typescript? It feels a bit cleaner.

Reply:
Sure :)